Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -129,6 +129,88 @@ void CruRawReader::dumpRDHAndNextHeader(const o2::header::RDHAny* rdh)
   LOG(info) << "######################### Finished RDH incoming buffer ##########################";
 }
 
+bool CruRawReader::checkRDH(const o2::header::RDHAny*  rdh)
+{
+// first check for bogus FEEID
+  TRDFeeID feeid;
+  feeid.word=0;
+  feeid.word= o2::raw::RDHUtils::getFEEID(rdh);
+  if (feeid.word==0xffff){
+    LOG(info) << "failed due to 0xffff : " << std::hex << feeid.word << " whole feeid : " << std::hex << (unsigned int)feeid.word;
```

Comment:
If I understood Guido correctly this would happen in case a CRU is not configured. And if everything is properly configured this should never happen. Thus I would throw an `error` here so that the operators see it immediately and call the TRD oncall. I assume we would want to configure the CRU correctly immediately when this happens, right?